recipe/build.sh

```diff
@@ -37,7 +37,7 @@ if [[ `uname` == 'Darwin' ]]; then
                 "${OPENBLAS_LIB}"
     done
 fi
-make test
+OPENBLAS_NUM_THREADS=$CPU_COUNT make test
```

Comment:
Don't think there were any issues with this, but felt like it was a good safety measure.